src/_data/site.js

```diff
@@ -1,5 +1,38 @@
 module.exports = {
-	title: "Fraunces",
+	title: "Fraunces by Undercase Type",
 	description:
-		'Fraunces is a display, "Old Style" soft-serif typeface inspired by the mannerisms of early 20th century typefaces such as Windsor, Souvenir, and the Cooper Series.'
+		'Fraunces is a display, "Old Style" soft-serif typeface inspired by the mannerisms of early 20th century typefaces such as Windsor, Souvenir, and the Cooper Series.',
+
+	// More info: https://css-tricks.com/essential-meta-tags-social-media/
+	metatags: [
+		{
+			property: "og:title",
+			content: "Fraunces by Undercase Type"
+		},
+		{
+			property: "og:description",
+			content:
+				"Fraunces is a display, 'Old Style' soft-serif typeface inspired by the mannerisms of early 20th century typefaces such as Windsor, Souvenir, and the Cooper Series."
+		},
+		{
+			property: "og:image",
+			content: "https://fraunces.undercase.xyz/img/fraunces_site.png"
```

Comment:
@RoelN just a heads-up that this is not going to work, as the image will be processed by webpack so the filename will be mangled. I think we should make the `webpackAsset` function available outside of templates to best resolve this. I'll try to pick it up later today.